frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -141,22 +145,29 @@ local VirtualKeyboard = InputContainer:new{
     umlautmode = false,
 
     width = Screen:scaleBySize(600),
-    height = Screen:scaleBySize(256),
-    bordersize = 2,
-    padding = 2,
+    height = nil,
+    bordersize = Screen:scaleBySize(2),
+    padding = Screen:scaleBySize(2),
     key_padding = Screen:scaleBySize(6),
 }
 
 local lang_to_keyboard_layout = {
     el = "el_keyboard",
     pl = "pl_keyboard",
     pt_BR = "pt_keyboard",
+    ja = "ja_keyboard",
```

Comment:
Oh, oops, no, I just added it in at the end without thinking.  I can fix that, too.